runtime/master/src/main/java/edu/snu/nemo/runtime/master/RuntimeMaster.java

```diff
@@ -186,7 +186,9 @@ public void terminate() {
         LOG.warn("Terminating master before all executor terminated messages arrived.");
       }
     } catch (final InterruptedException e) {
-      LOG.warn("Waiting executor terminating process interrupted.");
+      LOG.warn("Waiting executor terminating process interrupted: " + e);
+      // clean up state...
+      Thread.currentThread().interrupt();
```

Comment:
Hmm.. this patch is just for SonarQube patch...

Reply:
I don't wanna talk about this anymore.. but you should be aware that someone may misunderstood this change is required to multiple job submission.